tst/EnergyPlus/unit/Autosizing/CoolingWaterDesAirOutletTempSizing.unit.cc

```diff
@@ -54,11 +54,13 @@
 #include <EnergyPlus/DataHVACGlobals.hh>
 #include <EnergyPlus/DataSizing.hh>
 #include <EnergyPlus/Fans.hh>
+#include <EnergyPlus/FluidProperties.hh>
 
 namespace EnergyPlus {
 
 TEST_F(AutoSizingFixture, CoolingWaterDesAirOutletTempSizingGauntlet)
 {
+    FluidProperties::GetFluidPropertiesData(*state);
```

Comment:
should we do it like this or should we do `state->dataFluidProps->init_state();`?

Reply:
Good point, either method will work. I guess it's the developer's preference. On one hand I can see which function is being called, on the other hand developers should know what that means.

Reply:
The benefit of your suggestion is not having to include the header if it's missing.

Reply:
Nope, need to include the header either way. So that leaves it to preference.

Reply:
Ok, I saw all of your responses. I'm cool with them. We'll learn as we go.